messenger/custom-transport.rst

```diff
@@ -85,13 +87,9 @@ Here is a simplified example of a database transport::
             }
 
             // Mark the message as "handled"
-            $this->db->createQueryBuilder()
-                ->update('my_queue')
-                ->setValues([
-                    'handled' => true
-                ])
-                ->where('id = :id')
+            $this->db->createQuery('UPDATE my_queue SET handled = :true WHERE id = :id')
```

Comment:
I know it's common to use things like `handled = :true` ... but then it's weird to do `->setParameter('true', true)`  Should we change this to `handled = :is_handled` and `->setParameter('is_handled', true)` ?

Reply:
It's odd to see a parameter for a constant value, why not use `SET handled = TRUE` directly?